docs/deployment/openshift.mdx

```diff
@@ -3,3 +3,422 @@ title: "OpenShift"
+oc adm policy add-scc-to-user beeai-scc -z beeai-sa
+```
+
+### Step 4: Create Configuration File
+
+Create a values file with OpenShift-specific configuration:
+
+`openshift-config.yaml`:
+```yaml
+# OpenShift-specific settings
+serviceAccount:
+  create: false  # We created it manually
+  name: beeai-sa
+
+# Security context for OpenShift
+podSecurityContext:
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
+
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+
+# Agent pod security context
+agent:
+  podSecurityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+      - ALL
+    readOnlyRootFilesystem: false
+    runAsNonRoot: true
+
+# Service configuration (change from default NodePort to ClusterIP for OpenShift)
+service:
+  type: ClusterIP  # Use ClusterIP since OpenShift Routes will handle external access
+
+# If you want to include agents from the default catalog:
+externalRegistries:
+  public_github: "https://github.com/i-am-bee/beeai-platform@release-v0.2.0#path=agent-registry.yaml"
+
+# Your custom agents as docker images
+providers:
+  # e.g.
+  # - location: ghcr.io/i-am-bee/beeai-platform-agent-starter/my-agent:latest
+  - location: <docker-image-id>
+
+# Generate the encryption key:
+#  - using UV (https://docs.astral.sh/uv/getting-started/installation/)
+#   $ uv run --with cryptography python3 -c 'from cryptography.fernet import Fernet; print(Fernet.generate_key().decode())'
+#  - using python3 directly
+#   $ python3 -m pip install cryptography
+#   $ python3 -c 'from cryptography.fernet import Fernet; print(Fernet.generate_key().decode())'
+encryptionKey: "encryption-key-from-command"
+
+features:
+  uiNavigation: true  # Enable UI navigation (default is false)
+
+# Authentication configuration
+auth:
+  enabled: true
+  admin_password: "my-secret-password"
+
+# Help us improve the platform by sharing anonymized telemetry data
+telemetry:
+  sharing: true
+
+# PostgreSQL configuration for OpenShift
+postgresql:
+  enabled: true
+  auth:
+    enablePostgresUser: true
+    username: beeai-user
+    password: "password"
+    database: beeai
+  primary:
+    podSecurityContext:
+      runAsNonRoot: true
+      seccompProfile:
+        type: RuntimeDefault
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: false
+      runAsNonRoot: true
+```
+
+### Step 5: Install the Chart
+
+Install the chart in your OpenShift project:
+
+```bash
+helm install -f openshift-config.yaml beeai \
+  oci://ghcr.io/i-am-bee/beeai-platform/beeai-platform-chart/beeai-platform:latest \
+  -n beeai-platform
+```
+
+### Step 6: Create OpenShift Route
+
+Create a route to expose the BeeAI platform:
```

Comment:
Here we're showing how to create a route for openshift, but for kubernetes deployment it's not clear how you can access it. We don't have an ingress in the helm chart yet (it belongs there) but I'd at least mention that you need to create an ingress pointed to the beeai-platform-svc service